### ISSUE TITLE
Conflict between "padding-line-between-statements" and "lines-around-comment"

### ISSUE THREAD

**Issue by simkessy** [eslint/eslint#10763]
I'm trying to set up eslint with prettier and trying to configure how it handles comments and variable declaration spacing. But every time I save, it just toggles back and forth between the two rules. 

* **ESLint Version:** 5.2.0
* **Node Version:** 8.11.3
* **npm Version:** 5.6

**What parser (default, Babel-ESLint, etc.) are you using?** Google

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
module.exports = {
    extends: [
        "google",
        "prettier"
    ],
    "plugins": [
        "prettier"
    ],
    "env": {
        "es6": true,
        "browser": true,
        "node": true
    },
    rules: {
        "linebreak-style": [
            "error",
            "windows"
        ],
        "quotes": [
            "warn",
            "double",
            {
                "avoidEscape": true
            },
        ],
        "semi": [
            "error",
            "always"
        ],
        "no-var": 0,
        "require-jsdoc": 0,
        "max-len": 0,
        "padded-blocks": 0,
        "comma-dangle": ["error", "never"],
        "no-alert": 1,
        "no-console": [
            "error",
            {
                allow: ["debug", "warn", "error", "info"]
            }
        ],
        "no-debugger": 2,
        "curly": 2,
        "one-var": ["error", "never"],
        "no-unused-vars": [
            "warn",
            {
                "varsIgnorePattern": "PreSaveAction|[Ii]gnore"
            }
        ],
        "quote-props": [2, "as-needed"],
        "new-cap": 0,
        "guard-for-in": 0,
        "no-multiple-empty-lines": [
            "warn", {
                "max": 1
            }
        ],
        "prefer-spread": 0,
        "no-trailing-spaces": 1,
        "brace-style": 0,
        "camelcase": 1,
        "prettier/prettier": 2,
        "padding-line-between-statements": [
            "error",
            { blankLine: "always", prev: ["const", "let", "var"], next: "*"},
            { blankLine: "never",    prev: ["const", "let", "var"], next: ["const", "let", "var"]}
        ],
        "lines-around-comment": [
            "error",
            {
              "beforeBlockComment": true,
              "afterBlockComment": false,
              "beforeLineComment": true,
              "afterLineComment": false,
              "allowBlockStart": true,
              "allowBlockEnd": true,
              "allowObjectStart": true,
              "allowObjectEnd": true,
              "allowArrayStart": true,
              "allowArrayEnd": true
            }
          ]
    }
};
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
    var tasksGridInitializing = true;
    var inDashboard = false;
    // Default: Not in global board
    var globalBoard = false;
    var initialLoad = true;

    // For filtering to only projects user is on project team for NRG-547 NRG-699
    var usersProjects = [];
    var usersProjectFilterExpr;
```

```bash

```

**What did you expect to happen?**
Add space before comment

```
    var inDashboard = false;

    // comment 1
    var globalBoard = false;
    var initialLoad = true;

    // comment 2
    var usersProjects = [];
    var usersProjectFilterExpr;
```

**What actually happened? Please include the actual, raw output from ESLint.**
It just toggles back and forth between having a space and not having a space.

**Comment by sampotts** [comment]
It'd be awesome if `padding-line-between-statements` had an option for comments 👍
